web/src/components/watches/DownstreamWatchVersionDiff.jsx

```diff
@@ -154,13 +154,23 @@ class DownstreamWatchVersionDiff extends React.Component {
   }
 
   getPaths = (files) => {
+    const { app } = this.props;
+    const downstream = app.downstreams[0];
```

Comment:
can you check for downstreams length here and return an empty array if no downstreams are available?

Reply:
kk.